Applications/Spire/Include/Spire/Ui/StaticDropDownMenu.hpp

```diff
@@ -62,6 +62,10 @@ namespace Spire {
       //! Returns the currently selected item.
       virtual QVariant get_current_item() const;
 
+      //! True iff the next item should be activated when the list is shown.
+      //! using the down arrow key. Defaults to true.
```

Comment:
Fix the documentation for this. Remove the period after "shown" and this function has no default value, so remove "Defaults to true.".